src/swarm/index.ts

```diff
@@ -0,0 +1,3 @@
+export { runSwarmRepl } from './repl.js';
```

Comment:
somewhere we should note that the `/swarm` import is node.js-only?

Reply:
Can this be part of the package export conditions and work for TypeScript? What would you suggest in this kind of case?

Reply:
it's intentionally not documented yet. i expect this implementation to change substantially over the next few weeks.